internal/setup/setup.go

```diff
@@ -76,14 +88,22 @@ const (
 	UntilMutate PathUntil = "mutate"
 )
 
+type GenerateKind string
+
+const (
+	GenerateNone  GenerateKind = ""
+	GenerateState GenerateKind = "chisel-state"
+)
+
 type PathInfo struct {
```

Comment:
I think this is the perfect opportunity to disable comparing paths with `==`. Let's add:
```
type PathInfo struct {
	// Disable ==.
	_ [0]func()
```
at the top

Reply:
Done in b6bca94.